src/main/scala/com/github/unchama/seichiassist/subsystems/vote/domain/VoteCounter.scala

```diff
@@ -0,0 +1,5 @@
+package com.github.unchama.seichiassist.subsystems.vote.domain
+
+case class VoteCounter(value: Int) {
```

Comment:
```suggestion
case class VoteCount(value: Int) {
```

カウンターは意味が異なります (参考: [weblio](https://ejje.weblio.jp/content/counter))